README.md

```diff
@@ -81,3 +81,212 @@ The following options are specific to this plugin:
   include A/B testing or localization specific output. Note: if you
   invoke the callback more than once, you should change the output
   filename to avoid overwriting previous invocations' output.
+
+
+## Examples
```

Comment:
I'm on the fence about this. I know some projects take this approach of heavy upfront examples in the readme, but I'm not sure if it's a good fit here.

The examples are fairly verbose, and I think the fact that we're adding a whole `examples/` working directory reduces the need for the upfront examples.

You might consider having a `Quick Example` in the main README and pointing to the other more involved examples via `examples/README.md` and emphasize the different types of common workflows.

I think the information is good and valuable, I just question if it's too overwhelming (and intimidating) for someone jumping into a new eyeglass integration.